accel/tcg/translator.c

```diff
@@ -180,6 +205,13 @@ void translator_loop(const TranslatorOps *ops, DisasContextBase *db,
         }
     }
 
+#ifdef CONFIG_TCG_LOG_INSTR
+    /* Commit previous instruction */
+    if (unlikely(log_instr_enabled)) {
+        qemu_log_gen_printf_flush(db, true);
```

Comment:
Is this duplicated here because when exiting the TB we will get the state cleared before we get to the commit in the next TB first instruction above? If so maybe warrant a comment.

Reply:
That is the case.